Concurrency/AI_interface.py

```diff
@@ -380,3 +395,10 @@ def evaluate(self, config) -> None:
             time.sleep(1)
 
         ray.get(workers)
+
+
+if __name__ == '__main__':
```

Comment:
This should not be here. Just call the method from main and change the config to call the correct one.

Reply:
sure, but what about the comment I left? I thought that the `TFT_Vector_Pos_Simulator` was used in multiple places, but I just look and the only reference is inside the PPO method. Can I change remove the decorator and apply the "distributed computing wrapper" as a function inside the PPO method?